app/models/rent.rb

```diff
@@ -2,4 +2,18 @@ class Rent < ApplicationRecord
   belongs_to :user
   belongs_to :book
   validates :user, :book, :start_date, :end_date, presence: true
+
+  after_create :increase_times_rented
+
+  scope :are_active, lambda { |today|
+                       where("start_date < '#{today}' AND end_date > '#{today}' AND returned = false")
```

Comment:
If you need to interpolate values use a `placeholder`, it’s used for security to avoid `SQL injection attacks`
```ruby
where('start_date < ? AND end_date > ? AND returned = false', today)
#OR
where('start_date < :today AND end_date > :today AND returned = false', today: today)
```